### PR TITLE
Add components

### PR DESCRIPTION
**Pull request by razbakov**
Init project:
- add build tools and configuration

Add components:
- Editor component
- Field component
- Example app
- Test
- README

Add bundle

Closes [ch605]

---

You can review just [component source code](https://github.com/banovo/vue-form-wrapper/pull/1/commits/313658e8bc35def0b80d550edddfb44799192536)

### CODE REVIEW

File: build/utils.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        })
+      })
+    }
+
+    // Extract CSS when that option is specified
+    // (which is the case during production build)
+    if (options.extract) {
+      return ExtractTextPlugin.extract({
+        use: loaders,
+        fallback: 'vue-style-loader'
```
Comment (eFzvai) [inline build/utils.js]: Why you use in this case multiline?

File: dist/vue-form-wrapper.js [hunk 2]
```diff
@@ -0,0 +1,11647 @@
...
+		module.exports = factory();
+	else if(typeof define === 'function' && define.amd)
+		define([], factory);
+	else if(typeof exports === 'object')
+		exports["vue-form-wrapper"] = factory();
+	else
+		root["vue-form-wrapper"] = factory();
+})(this, function() {
+return /******/ (function(modules) { // webpackBootstrap
+/******/ 	// The module cache
```
Comment (eFzvai) [inline dist/vue-form-wrapper.js]: Why this comments?

**Comment by razbakov** [comment]
You review [generated code](https://github.com/vuejs-templates/webpack). As I noticed in PR it makes sense to review just my [component source code](https://github.com/banovo/vue-form-wrapper/pull/1/commits/313658e8bc35def0b80d550edddfb44799192536)

File: dist/vue-form-wrapper.js (hunk 2 above)
Comment (razbakov) [reply dist/vue-form-wrapper.js]: it is generated bundle

Comment (eFzvai) [reply dist/vue-form-wrapper.js]: Where are the files come from?

Comment (razbakov) [reply dist/vue-form-wrapper.js]: `npm run bundle` - it generates assets: `dist/*.js`, `dist/*.min.js`, `dist/*.css`.
Those files have to be in repo, so you can require them in `package.json` in other apps

Comment (eFzvai) [reply dist/vue-form-wrapper.js]: If this files are generated we don't need them in our repository, we can generate it if some component require this files.

Comment (razbakov) [reply dist/vue-form-wrapper.js]: we can't, this is hone npm needs it

**Pull request merged**